pkg/cmd/checkendpoints/controller/pod_network_connectivity_check_controller.go

```diff
@@ -97,6 +107,27 @@ func (c *controller) Sync(ctx context.Context, syncContext factory.SyncContext)
 	return nil
 }
 
+// getClientCerts returns the client cert specified in the secret specified in the PodNetworkConnectivityCheck
+// or nil if not specified or there is an error retrieving the certs.
+func (c *controller) getClientCerts(check *operatorcontrolplanev1alpha1.PodNetworkConnectivityCheck) CertificatesGetter {
+	return func() []tls.Certificate {
```

Comment:
my advice about this signature was bad, wasn't it?  Should have been `func (check *operatorcontrolplanev1alpha1.PodNetworkConnectivityCheck) []tls.Certificate` and you could just directly reference eh?

Can be fixed up afterwards.